lib/tasks/smaxtec_api.rake

```diff
@@ -4,4 +4,10 @@ namespace :smaxtec_api do
     smaxtec_api_controller = SmaxtecApi.new
     puts smaxtec_api_controller.get_temperature
   end
+
+  desc "get sensor readings"
+  task :get_sensor_readings => :environment do
```

Comment:
Usually I try to avoid these all to generic `get_... ` or `set_... ` methods in favour of a more self-explaining name.